src/ChunkDef.h

```diff
@@ -112,7 +132,7 @@ class cChunkDef
 	/** Converts relative block coordinates into absolute coordinates with a known chunk location */
 	inline static Vector3i RelativeToAbsolute(const Vector3i & a_RelBlockPosition, int a_ChunkX, int a_ChunkZ)
 	{
-		return Vector3i(a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width);
+		return {a_RelBlockPosition.x + a_ChunkX * Width, a_RelBlockPosition.y, a_RelBlockPosition.z + a_ChunkZ * Width};
```

Comment:
If replacing `const Vector3i &` in the param list with `Vector3i` works without further changes to the code, consider doing so now. If it requires other changes, let's leave it into another PR.